tikv-client/src/main/java/com/pingcap/tikv/codec/CodecDataInput.java

```diff
@@ -15,11 +15,11 @@
 
 package com.pingcap.tikv.codec;
 
+import com.google.common.io.LittleEndianDataInputStream;
 import com.google.protobuf.ByteString;
 import java.io.*;
 import java.util.Arrays;
 import javax.annotation.Nonnull;
-import parquet.bytes.LittleEndianDataInputStream;
```

Comment:
why change package name?

Reply:
It was a mistake to use parquet's package in client.

Reply:
how can we avoid this kind of error in the future?

Reply:
since we have moved the spark dependencies into core, we will not be able to introduce incorrect package names in spark in future.

Reply:
since we have moved the spark dependencies into core, we will not be able to introduce incorrect package names in spark in future.